assets/css/style.css

```diff
@@ -123,7 +120,7 @@ label {
     font-style: normal;
     font-weight: 500;
     font-size: 1.042vw;
-    line-height: 1.389vw;
+    line-height: 1.389;
     color: #344563;
```

Comment:
- [ ] Please align your text center as the Figma design.
## your design 👇
![langstdesk](https://user-images.githubusercontent.com/66992901/193132542-444790c6-48e2-48dd-ae03-6d557ac88cac.PNG)

## Figma design👇
![langfigmadesk](https://user-images.githubusercontent.com/66992901/193132641-d522cd37-8040-4539-b81e-7f4643dca2a9.PNG)